Sources/AuthFoundation/Network/APIResponse.swift

```diff
@@ -29,39 +29,47 @@ public struct APIResponse<T: Decodable>: Decodable {
     public let links: [Link: URL]
     
     /// Information about the current rate limit.
-    public let rateInfo: RateLimit?
+    public let rateInfo: ApiRateLimit?
```

Comment:
This looks like a breaking API change. Is it possible to do this without the rename? I don't think we'd want to break API compatibility for this change.

Reply:
I think this is a safe enough change, because there's no public initializers for this type, so it's only ever accessed as a property on the APIResponse object.

Reply:
I think we should take a stance here. If we're not adhering to semver, we need to document it. Oktas default is semver.

Reply:
Docs stating semver. https://developer.okta.com/code/library-versions/#semantic-versioning

Reply:
@emmanuelogunmuyiwa-okta can you look into whether or not we can add some sort of typealias or other value with an `@available` depreciation warning to prevent needing to bump the major version?

Reply:
@mikenachbaur-okta I think including the depreciation warning this way might be better that way its easier to adopt.  

```
@available(*, unavailable, renamed: "APIRateLimit")
public struct RateLimit { }
```

Reply:
Yes, but if you use the following, it won't require the user to make any code changes, which ensures that no breaking changes will occur:

```
public struct APIResponse<T: Decodable>: Decodable {
    @available(*, deprecated, renamed: "APIRateLimit")
    public typealias RateLimit = APIRateLimit
```

Reply:
That sounds good. If I use the typealias, I would not add the depreciation warning because Xcode would require a code change with that.